projects/kubernetes/e2e/e2e.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+kubectl get namespaces \
+  --output="jsonpath={range .items[?(.status.phase == \"Active\")]}{.metadata.name}{\"\n\"}{end}" \
+  | grep '^e2e-.*' \
+  | xargs -r kubectl delete namespaces
+
+exec /usr/bin/ginkgo \
+  -nodes=4 \
+  -progress \
+  -flakeAttempts=2 \
+  -skip='\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[HPA\]|Dashboard|Services.*functioning.*NodePort|.*NFS.*|.*Volume.*|\[sig-storage\]|.*StatefulSet.*|should\ proxy\ to\ cadvisor\ using\ proxy\ subresource' \
+  /usr/bin/e2e.test -- \
+    -provider=local \
+    -num-nodes=1 \
+    -kubeconfig=/etc/kubernetes/admin.conf \
+    -host=https://192.168.65.2:6443 \
```

Comment:
Ah, I guess this IP is a dependency on desktop.

Reply:
Yeah it would be nice to make this not just work on desktop, but we can always do later potentially.